src/common/injectGlobals.js

```diff
@@ -19,9 +19,12 @@ if (personaApiOverride) {
   global.log('warning: using persona api override');
 }
 
+export const arrangerProjectId = process.env.REACT_APP_PROJECT_ID;
```

Comment:
wanna replace `process.env.REACT_APP_PROJECT_ID` with this in FileRepo.js?

Reply:
I don't follow. This is a global constant from our .env file, why would it be setup differently than the other .env variables?

Reply:
we are using ` process.env.REACT_APP_PROJECT_ID` directly in FileRepo.js,  just now that you added it here you could update it to use this `arrangerProjectId` value instead.

Reply:
OH, understood now. Will update.